src/main/java/com/jcabi/log/Logger.java

```diff
@@ -429,6 +429,32 @@ public void write(final int data) throws IOException {
     }
 
     /**
+     * Log messages constructed from Suppliers (Java 8 users).
+     * It is convenient to use method referencing because the method
+     * won't be called unless the specified logging level is enabled.
+     *
+     * This saves you the effort of having to check if the level is enabled
+     * before calling the logging method.
+     * E.g.
+     * <pre>
+     *     if (Logger.isDebugEnabled(this)) {
+     *         Logger.debug(this, "Some %s", calculate());
+     *     }
+     * </pre><br>
```

Comment:
@mkordas it looks better in the parsed Javadoc (when you hover your mouse over the method call)